shell/common/skia_event_tracer_impl.cc

```diff
@@ -35,13 +37,10 @@
 #define TRACE_VALUE_TYPE_COPY_STRING (static_cast<unsigned char>(7))
 #define TRACE_VALUE_TYPE_CONVERTABLE (static_cast<unsigned char>(8))
 
-#endif  // defined(OS_FUCHSIA)
-
 namespace flutter {
 
 namespace {
 
-#if defined(OS_FUCHSIA)
 template <class T, class U>
```

Comment:
If you're going to include |BitCast| in non-Fuchsia builds, can you make sure that the "#include \<algorithm\>" and "#include \<cstring\>" lines are also included in non-Fuchsia builds?  You should be able to just move them adjacent to the existing "#include \<vector\>" line.

Reply:
Maybe '<algorithm>' and '<cstring>' don't necessarily need include? The build tests have passed.